src/build/cargo.rs

```diff
@@ -299,20 +308,29 @@ impl Executor for RlsExecutor {
             }
         }
 
-        if self.workspace_mode {
-            let output = cmd.output().expect("Couldn't execute rustc");
-            let mut stderr_json_msg = convert_message_to_json_strings(output.stderr);
-            self.compiler_messages.lock().unwrap().append(&mut stderr_json_msg);
-        } else {
-            cmd.status().expect("Couldn't execute rustc");
-        }
-
         // Finally, store the modified cargo-generated args/envs for future rustc calls
         args.insert(0, rustc_exe);
```

Comment:
Oh, okay, I thought `run_compiler` accepts args with the first program argument omitted, nevermind then :+1: